app/lib/actions/katello/repository/clone_to_version.rb

```diff
@@ -20,6 +20,8 @@ def plan(repository, content_view_version, incremental = false)
               plan_action(Repository::CloneDockerContent, repository, new_repository)
             elsif new_repository.ostree?
               plan_action(Repository::CloneOstreeContent, repository, new_repository)
+            elsif new_repository.file?
+              plan_action(Repository::CloneFileContent, repository, new_repository)
```

Comment:
This seems to be the reason why you weren't seeing directories created for the lifecycle environments.  After adding this logic to that file it seems to publish those directories.